src/main/java/julia/git/Commit.java

```diff
@@ -0,0 +1,46 @@
+package julia.git;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class Commit implements Listable {
```

Comment:
I don't think it will make sense to make it Listable - it's not a directory. So you can't get a list of inside objects.